WebAssets/tests/hints.tests.ts

```diff
@@ -1,8 +1,14 @@
-const TestDriver = require('./test-driver.js');
+import type { CompletionItemData } from '../ts/interfaces/protocol';
+import { TestDriver } from './test-driver';
 
 // TODO: remove in year 3000 when TC39 finally specs this
 // eslint-disable-next-line no-extend-native
-Array.prototype.last = Array.prototype.last || function() { return this[this.length - 1]; };
+(Array.prototype as any).last = (Array.prototype as any).last || function<T>(this: Array<T>) { return this[this.length - 1]; };

```

Comment:
Why is it here too? Thought I was in a dejavu

Reply:
Yep, initial one was one line so it was easier to copy over than reuse in a module.
Tbh probably need to use something like https://github.com/tc39/proposal-array-last/issues/19#issue-344197727 instead for now as there was no progress on `last` for a while.